app/policies/comment_policy.rb

```diff
@@ -0,0 +1,37 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+
+  def initialize(user, comment)
+    @user = user
+    @comment = comment
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    false
+  end
+
+  def new?
+    true
+  end
+
+  def edit?
+    user == comment.author
```

Comment:
the issue referenced in the screenshots is probably that this should be `@comment` rather than `comment`